shells/chrome/devtools-init.js

```diff
@@ -0,0 +1,8 @@
+chrome.devtools.panels.create(
+  "Datalog DB",
+  "",
+  "inspect-panel.html",
+  function (panel) {
+    console.log("panel initialized", panel);
```

Comment:
should this be removed or made more explicit like “datalog-console panel initialized”?

Reply:
I went to remove this but discovered some panel listeners which may be meaningful for more granular control when the panel is subscribing to updates. So there is a new placeholder here now and we can discuss.